pydash/functions.py

```diff
@@ -210,6 +211,32 @@ def callback(item):
         return pyd.some(obj, callback)
 
 
+def flip(func):
+    """Creates a function that invokes the method with arguments reversed.
+
+    Args:
+        func (function): Function to flip arguments for.
+
+    Returns:
+        function: Returns the new flipped function.
+
+    Example:
+
+        >>> flipped = flip(lambda args: args)
+        >>> flipped(1, 2, 3, 4)
+        (4, 3, 2, 1)
+        >>> flipped = flip(lambda args: [i * 2 for i in args])
+        >>> flipped(1, 2, 3, 4)
+        [8, 6, 4, 2]
+
+    .. versionadded:: TODO
+    """
+    def _flip(*args):
+        return func(args)[::-1]
```

Comment:
This is supposed to reverse the arguments given, not the result. And it's not supposed to convert `*args` into a tuple.

Should be defined more like:

```python
def _flip(*args, **kargs):
    return func(*reversed(args), **kargs)
```

Also, instead of defining as an inline function, should go ahead and define it as a class similar to how all the other functions in this module are defined (for consistency).